django/tasks/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .base import Task, get_backend  # NOQA
+from .backends import TaskResult  # NOQA
+from .constants import *  # NOQA
+from .register import registry
+
+
+def task(name_or_func=None, name=None, using=None, options=None):
+    # @task
+    if callable(name_or_func):
+        if isinstance(name_or_func, Task):
+            name_or_func = name_or_func.run
```

Comment:
If given a `Task` subclass, why do we create a new `Task` using its `run` method, rather than just using the `Task` class we were given?